app/packages/registration-schema/validations.ts

```diff
@@ -99,6 +99,23 @@ export function validateNodeLicense() {
     };
 }
 
+export function validateSubjects() {
+    return async (_: unknown, __: unknown, ___: unknown, ____: unknown, content: DraftRegistration) => {
+        const subjects = await content.subjects;
```

Comment:
Does this resort to sending a request every time we validate MetadataChangeset? If yes, this would result in multiple extra requests every time the user switches between pages as per https://github.com/CenterForOpenScience/ember-osf-web/pull/965/files#diff-d0f667f45e345771e2f5abe1f08e38d6R34

Also, are subjects paginated? This only returns the first page.

Reply:
This will send a request very time we validate the changeset. I don't know whether it is paginated, but it shouldn't matter in this scenario as we are only checking for whether there is a subject.

Reply:
The issue I see is that if the user had more than one page of subjects on the draft-registration, this request will change the relationship data in the data store to only have 10.
It's worth checking if this scenario is correct.

Reply:
It will, but it should not matter. When the user navigates back to the metadata page, the subject picker will make sure all the subjects are loaded.